### PR TITLE
Create backend functionality to remove users with expired hearbeats

### PR DESCRIPTION
**Pull request by iva-stoycheva**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+import {heartbeatInterval} from "./constants.js";
 
 const app = express();
 app.use(express.json());
 
 const dynamo = new Dynamo();
 
+setInterval(async () => {
+    await dynamo.deleteStaleUsers();
+    }, heartbeatInterval);
```
Comment (hsiliev) [inline src/app.js:12]: minor (formatting): Shouldn't this be in the beginning of the line

File: src/dynamo.js [hunk 2]
```diff
@@ -37,7 +37,7 @@ export class Dynamo {
             })
             );
         let params = { RequestItems: { OnlineUsers: userDeleteRequests } };
-        await client.batchWriteItem(params).catch(e => console.log(e));
+        await client.batchWriteItem(params).catch(() => console.log('All expired users are deleted'));
```
Comment (hsiliev) [inline src/dynamo.js:40]: Is this true? On catch it might be that not all are deleted?

Comment (SlavenGrowy) [reply src/dynamo.js:40]: Yep, this is broken. I had a call with @iva-stoycheva how this can be fixed

**Review by SlavenGrowy — CHANGES_REQUESTED** [review]

**Review by SlavenGrowy — APPROVED** [review]
LGTM 🌳

**Pull request merged**